generate_build_config.py

```diff
@@ -33,23 +34,43 @@
 - mv $CHROOT_ROOT/build/livecd.ubuntu-cpc.* /home/ubuntu/images
 """
 
+WRITE_FILES_TEMPLATE = """\
+write_files:
+- encoding: b64
```

Comment:
And this can be another PR.  My abstention for this PR is the example customization script comments only.